### ISSUE TITLE
Add health endpoint

### ISSUE THREAD

**Issue by Mortinke** [logzio/logzio-docker#19]
Currently, we trust that the logs will be shipped to logzio. However, it would be nice, if a health endpoint could be added for the logzio-docker container, so we can check the process.

**Comment by idohalevi** [comment]
@Mortinke 
Good idea, we will look into it. In what context are you using logzio-docker?

**Comment by Mortinke** [comment]
@idohalevi thanks for the fast response! We use _logzio-docker_ to forward the docker logs of the containers from our ECS clusters to logz.io.
